lib/hammer_cli_foreman/location.rb

```diff
@@ -24,6 +24,7 @@ class InfoCommand < HammerCLIForeman::InfoCommand
       option '--id', 'ID', _("Location numeric id to search by")
 
       output ListCommand.output_definition do
+        field :parent_name, _("Parent")
```

Comment:
This could as well be:
```ruby
field nil, _("Parent"), Fields::SingleReference, :key => :parent
```
which would enable parent ids to be shown when `--show-ids` is used:
```
Parent:   Default Location (id: 2)
```